### DISCUSSION TITLE
Deploy using helm without zone-aware replication and any PVC

### DISCUSSION THREAD

**Discussion by libormezl** [grafana/mimir#9118] [Help and support]
Hi, I am quite new to the LGTM stack. I have successfully deployed Grafana with Tempo to our AKS cluster, but I am having some issues with Mimir.

I am deploying using Helm. My goal is to deploy Mimir in microservice mode without zone-aware replication (we don't need it) and **without using any PVCs**, as I want to use Azure Storage as the storage solution.

I have disabled _zoneAwareReplication_ and also set _persistentVolume.enabled=false_ for all services in my Helm configuration. I have set up Azure Storage (it seems to be working).

However, I am encountering two issues with the configuration:

- When I query from Grafana, I get _too many unhealthy instances in the ring_.
- When I push data from the OTEL Collector, the collector reports responded with _HTTP Status Code 503, Message=send data to ingesters: at least 1 live replica required, could only find 0_

Is it possible to achieve my goal, or is it necessary to have PVCs?

**Comment by pstibrany** [comment]
PVCs are required for ingesters. PVCs are not required, but strongly recommended for store-gateways, to avoid slow startup. PVC is also recommended for compactors, to being able to guarantee requested iops.
